app/store/services/helpers/gps.js

```diff
@@ -25,20 +25,19 @@ const isInArea = (
   Math.abs(position.longitude - point.longitude) < Δλ &&
   Math.abs(position.latitude - point.latitude) < Δφ;
 
-const getEventDistance = (points = []) => {
+const computePathDistance = (points = []) =>
```

Comment:
merge this function with const computeDistance 
using the following signature
(...nodes = [])=>{
//compute it here and return
}

Reply:
rest parameters should not be initialized!!